src/change/sdp.rs

```diff
@@ -879,12 +887,13 @@ fn update_media(media: &mut Media, m: &MediaLine, config: &mut CodecConfig, stre
     for rid in m.rids().iter() {
         media.expect_rid(*rid);
     }
+    dbg!(m, &config);
 
     // Narrowing/ordering of of PT
     let pts: Vec<Pt> = m
         .rtp_params()
         .into_iter()
-        .filter(|p| config.matches(p))
+        .filter_map(|p| config.match_params(p))
```

Comment:
This also isn't quite right, we should attempt to match the Pt used by the offerer if we haven't already locked down the codec in question.

If the offer specifies `120=VP8/90000` and we have a default of `96=VP8/90000` this will use our default in the case when we haven't previous locked down `VP8/90000`, we should respect the offerer's suggestion